app/assets/javascripts/browsered/web-payments/apple-pay.js

```diff
@@ -70,5 +70,16 @@ module.exports = () => {
     })
   }
 
+  session.onshippingcontactselected = function (event) {
```

Comment:
1. is this method called by magic?
2. does `onshippingcontactselected` need to all be in lower case?

Reply:
Yeah https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession/1778008-completeshippingcontactselection